canonical-web-and-design/ubuntu.com
# Add mosquitto appliance page

## Done

- Add mosquitto appliance page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/mosquitto
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the data in the [copy doc](https://docs.google.com/document/d/1Gg4Ltt550H8zN4dmZpd5cjIeRz52AyXFL-nDrGI5rb8/edit?usp=sharing)
